specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-03-02-preview/managedClusters.json

```diff
@@ -5897,6 +5915,40 @@
       },
       "description": "Workload Identity settings for the security profile."
     },
+    "ManagedClusterStorageProfileDiskCSIDriver": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether to enable AzureDisk CSI Driver"
+        },
+        "version": {
+          "type": "string",
+          "description": "the version of AzureDisk CSI Driver"
```

Comment:
Better capitalize and add `.` at the end.

Reply:
done